inputfiles/idl/Permissions.widl

```diff
@@ -46,6 +46,8 @@ enum PermissionName {
   "gyroscope",
   "magnetometer",
   "clipboard",
+  "clipboard-read",
+  "clipboard-write",
```

Comment:
According to the MDN `clipboard-read` and `clipboard-write` are the correct permission names.

https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query#Parameters

Reply:
Permissions names are right, it's IDL that does not match the spec one. I filed https://github.com/w3c/clipboard-apis/issues/101 to update the spec so that we can use them correctly.